sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -237,6 +246,99 @@ def get_custom_model(self, model_id, **kwargs):
+        # type: (str, str, Any) -> Dict[str, Union[str, int]]
+        """Generate authorization for copying a model into the target Form Recognizer resource.
+        This should be called by the target resource (where the model will be copied to)
+        and the output can be passed as the `target` parameter into :func:`~begin_copy_model()`.
+
+        :param str resource_id: Azure Resource Id of the target Form Recognizer resource
+            where the model will be copied to.
+        :param str resource_region: Location of the target Azure resource. A valid Azure
+            region name supported by Cognitive Services.
+        :return: A dictionary with values for the copy authorization -
+            "modelId", "accessToken", "resourceId", "resourceRegion", and "expirationDateTimeTicks".
+        :rtype: Dict[str, Union[str, int]]
+        :raises ~azure.core.exceptions.HttpResponseError:
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_copy_model.py
+                :start-after: [START authorize_copy_target]
+                :end-before: [END authorize_copy_target]
+                :language: python
+                :dedent: 8
+                :caption: Authorize the target resource to receive the copied model
+        """
+
+        response = self._client.generate_model_copy_authorization(  # type: ignore
+            cls=lambda pipeline_response, deserialized, response_headers: pipeline_response,
+            error_map=error_map,
+            **kwargs
+        )  # type: PipelineResponse
+        target = json.loads(response.http_response.text())
+        target["resourceId"] = resource_id
+        target["resourceRegion"] = resource_region
+        return target
+
+    @distributed_trace
+    def begin_copy_model(
+        self,
+        model_id,  # type: str
+        target,  # type: Dict
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> LROPoller
+        """Copy a custom model stored in this resource (the source) to the user specified
+        target Form Recognizer resource. This should be called with the source Form Recognizer resource
+        (with the model that is intended to be copied). The `target` parameter should be supplied from the
+        target resource's output from calling the :func:`~authorize_copy_target()` method.
+
+        :param str model_id: Model identifier of the model to copy to target resource.
+        :param dict target:
+            The copy authorization generated from the target resource's call to
+            :func:`~authorize_copy_target()`.
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if
+            no Retry-After header is present.
+        :return: An instance of an LROPoller. Call `result()` on the poller
+            object to return a :class:`~azure.ai.formrecognizer.CustomFormModelInfo`.
+        :rtype: ~azure.core.polling.LROPoller[CustomFormModelInfo]
```

Comment:
nit: I think you would need ~azure.core.polling.LROPoller[~azure.ai.formrecognizer.CustomFormModelInfo]